### PR TITLE
Releases

### PR DESCRIPTION
**Pull request by j-mendez**

### CODE REVIEW

File: app/controllers/bootstrap.js [hunk 1]
```diff
@@ -1,53 +1,44 @@
...
-
+myApp.controller('AccordionDemoCtrl', function ($scope, $http, $timeout, $rootScope, $location) {
+var parent = $rootScope;
+	$scope.status = {
+	    isFirstOpen: true,
+	    isFirstDisabled: false
+	  };
+    $scope.validateClick = function (group, index) {
+        if (group.correct == "addingDegreeCourse") {
+            group.isaddingDegreeCourse = true;
```
Comment (reactivepixel) [inline app/controllers/bootstrap.js:9]: When using camelCase ensure that you strictly adhere to camelCasing

group.is**A**ddingDegreeCourse

File: app/controllers/bootstrap.js [hunk 2]
```diff
@@ -1,53 +1,44 @@
...
+	      	{"value": "Show Production"}
+	      ],
+	      type:"select",
+	      name:"major",      
 	    },
 	    { 
 	      // label will render the label for the input form.
 	      label: 'Course Name',
-	      placeholder: ' Enter Full Name ',
+	      placeholder: ' Enter course name ',
```
Comment (reactivepixel) [inline app/controllers/bootstrap.js:38]: Should _Course Name_ be Capitalized?

Comment (j-mendez) [inline app/controllers/bootstrap.js:38]: Yes It should will fix it now thanks

File: app/controllers/bootstrap.js [hunk 3]
```diff
@@ -99,105 +92,78 @@ myApp.controller('AccordionDemoCtrl', function ($scope, $http, $timeout) {
...
+  	$scope.submitForm = function($scope){
+
+        //parcing the data from the form
+        var major = JSON.stringify($rootScope.groups[0].users[0].value);
+        var course = $rootScope.groups.course;
+        var courseDescript = $rootScope.groups.description;
+        var rubName = $rootScope.groups.name;
+        var secTitles = $rootScope.groups.title;
+        var grade = $rootScope.groups.grade;
+        var rDescript = $rootScope.groups.rubricdescription;
```
Comment (reactivepixel) [inline app/controllers/bootstrap.js:117]: Breaking camelCasing with `rubricdescription`

File: app/controllers/bootstrap.js [hunk 4]
```diff
@@ -99,105 +92,78 @@ myApp.controller('AccordionDemoCtrl', function ($scope, $http, $timeout) {
...
+        var due = $rootScope.groups.dueDate;
+
+		//this the first form that gets submited   
+  		console.log('FIRED', JSON.stringify($rootScope.groups[0].users[0].value));
```
Comment (reactivepixel) [inline app/controllers/bootstrap.js:121]: Testing comments should be suppressed

File: app/controllers/bootstrap.js [hunk 5]
```diff
@@ -99,105 +92,78 @@ myApp.controller('AccordionDemoCtrl', function ($scope, $http, $timeout) {
...
+        //seperating the grade and sectitles on commas
+        var sections = secTitles;
+        var sectionSplit = sections.split(",");
+        console.log(sectionSplit);
+        var grades = grade;
+        var gradeSplit = grades.split(",");
+        console.log(gradeSplit);
+        
+        //this the first form that gets submited
+       var cData = { major: major, course: course ,description: courseDescript, rubric:{name:rubName,title:sectionSplit, grade:gradeSplit, dueDate: due,rubricdescription:rDescript}};
```
Comment (reactivepixel) [inline app/controllers/bootstrap.js:136]: `cData` is not a descriptive Variable Name

**Pull request merged**